src/index.js

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.scss';
-import App from './components/container/App/App';
+import './index.css';
+import App from './App';
+import Cards from './componnts/index';
+import reportWebVitals from './reportWebVitals';
+
+
+
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
+    <Cards />
```

Comment:
I'm afraid this file also needs to be reverted to the original state

Reply:
what @tinkabel85 said